### PR TITLE
New overview section

### PR DESCRIPTION
**Pull request by chasulin**
`Location page redesign/Location overview` in storybook

### CODE REVIEW

File: src/components/LocationPage/ChartsHolder.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import { ChartContentWrapper } from './ChartsHolder.style';
 import { useProjectionsFromRegion } from 'common/utils/model';
 import { LoadingScreen } from 'screens/LocationPage/LocationPage.style';
 import LocationPageHeader from './LocationPageHeader';
-import { summaryToStats } from 'components/NewLocationPage/SummaryStatsBlock/utils';
+import { summaryToStats } from 'components/NewLocationPage/SummaryStat/utils';
```
Comment (chasulin) [inline src/components/LocationPage/ChartsHolder.tsx:26]: renamed the whole summary stat folder + the files inside, now that its no longer a block // the folder is just the code for the individual stat

File: src/components/NewLocationPage/VaccinationProgressBarBlock/ProgressBar.tsx [hunk 2]
```diff
@@ -1,20 +1,16 @@
 import React from 'react';
 import { ParentSize } from '@vx/responsive';
 import { v4 as uuidv4 } from 'uuid';
-import { ProgressBarContainer } from './VaccinationProgressBar.style';
+import { ProgressBarContainer } from './VaccinationProgressBarBlock.style';
```
Comment (chasulin) [inline src/components/NewLocationPage/VaccinationProgressBarBlock/ProgressBar.tsx:4]: also renamed the folder + files of the progress bar

File: src/components/NewLocationPage/VaccinationProgressBarBlock/VaccinationProgressBarBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { ProgressBar } from './ProgressBar';
+import { Title } from './VaccinationProgressBarBlock.style';
+
+export interface ProgressBarProps {
+  vaccinationsInitiated: number;
+  vaccinationsCompleted: number;
+  locationName: string;
+}
+
+const VaccinationProgressBarBlock: React.FC<ProgressBarProps> = ({
```
Comment (chasulin) [inline src/components/NewLocationPage/VaccinationProgressBarBlock/VaccinationProgressBarBlock.tsx:11]: this file is the addition of the 'vaccination progress' title

File: src/components/NewLocationPage/VaccinationProgressBarBlock/ProgressBar.tsx [hunk 4]
```diff
@@ -41,19 +37,19 @@ const ProgressBar: React.FC<{
       <defs>
         <linearGradient id={gradientId}>
           <stop
-            offset={getOffsetPercentage(vaccinationsInitiated)}
+            offset={getOffsetPercentage(vaccinationsCompleted)}
```
Comment (chasulin) [inline src/components/NewLocationPage/VaccinationProgressBarBlock/ProgressBar.tsx:40]: i realized i had these backwards

Comment (pnavarrc) [reply src/components/NewLocationPage/VaccinationProgressBarBlock/ProgressBar.tsx:40]: 🙃

File: src/components/NewLocationPage/LocationOverview/LocationOverview.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import { Metric } from 'common/metricEnum';
+
+const LocationOverview: React.FC<{
+  region: Region;
+  locationSummary: LocationSummary;
+}> = ({ region, locationSummary }) => {
+  const stats = summaryToStats(locationSummary);
+
+  return (
+    <SectionContainer style={{ border: '1px dotted gray' }}>
```
Comment (chasulin) [inline src/components/NewLocationPage/LocationOverview/LocationOverview.tsx:27]: dotted border for QA

File: src/components/NewLocationPage/LocationOverview/LocationOverview.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+import { Metric } from 'common/metricEnum';
+
+const LocationOverview: React.FC<{
+  region: Region;
+  locationSummary: LocationSummary;
+}> = ({ region, locationSummary }) => {
+  const stats = summaryToStats(locationSummary);
+
+  return (
+    <SectionContainer style={{ border: '1px dotted gray', maxWidth: '880px' }}>
```
Comment (chasulin) [inline src/components/NewLocationPage/LocationOverview/LocationOverview.tsx:27]: for QA: 880px is the true max width of the section containers in the mocks. also dotted border for qa

**Review by pnavarrc — APPROVED** [review]
Looks great! 👏 👏 👏

**Pull request merged**